pkgs/os-specific/darwin/xcode/default.nix

```diff
@@ -1,10 +1,10 @@
-{ stdenv, requireFile, lib }:
+{ requireFile, targetPlatform, lib }:
 
 let requireXcode = version: sha256:
   let
     xip = "Xcode_" + version +  ".xip";
     # TODO(alexfmpe): Find out how to validate the .xip signature in Linux
-    unxip = if stdenv.isDarwin
+    unxip = if targetPlatform.isDarwin
```

Comment:
this should be stdenv or buildPlatform ~ users will use these instructions on the machine their building things, not what they're targeting